packages/config/index.js

```diff
@@ -4,34 +4,68 @@ var path = require('path');
 
 var root = require('pkg-dir').sync(process.cwd());
 
-var npmConfig = require('./lib/parse-env')('hops');
-if (!Object.keys(npmConfig).length) {
-  try {
-    require.resolve(path.join(root, 'package.json'));
-    npmConfig = require(path.join(root, 'package.json')).config.hops;
-  } catch (_) {}
+var explorer = require('cosmiconfig')('hops', {
+  rcExtensions: true,
+  stopDir: root,
+  sync: true,
+});
+
+function getRawConfig() {
+  var result = explorer.load(process.cwd());
+  return Object.assign(
+    {
+      https: false,
+      host: '0.0.0.0',
+      port: 8080,
+      locations: [],
+      basePath: '',
+      assetPath: '',
+      browsers: '> 1%, last 2 versions, Firefox ESR',
+      node: 'current',
+      envVars: { HOPS_MODE: 'dynamic' },
+      moduleDirs: [],
+      appDir: '.',
+      buildDir: 'build',
+      cacheDir: 'node_modules/.cache/hops',
+    },
+    result ? result.config : {}
+  );
 }
 
-function extendConfig(config) {
-  if (npmConfig.extends) {
+function applyExtension(config) {
```

Comment:
I liked the old name `extendConfig` better, because `extension` is also used for file extensions, for example and it isn't clear when just reading the name of this function, what it actually does.

Reply:
I have now renamed  it to `applyExtensionConfig()` - does that work for you?